src/runtime.cc

```diff
@@ -46,28 +44,32 @@ namespace detail {
 		return *instance;
 	}
 
-	auto get_pid() {
+	static auto get_pid() {
 #ifdef _MSC_VER
 		return _getpid();
 #else
 		return getpid();
 #endif
 	}
 
-	const char* get_build_type() {
+	static std::string get_version_string() {
```

Comment:
Not really related to these changes, but do we want to expose this string to the user as well?

Reply:
I don't know, do we? We expose the macros, so they could always build the string pretty easily themselves.

Reply:
Fair enough!